packages/manager/apps/web/client/app/hosting/cdn/order/hosting-cdn-order.routing.js

```diff
@@ -185,18 +182,17 @@ export default /* @ngInject */ ($stateProvider) => {
         const {
           data: addonPlans,
         } = await HostingCdnSharedService.getCatalogAddonsPlan(serviceId);
-        const { data: addonPlan } = find(addonPlans, ({ planCode }) =>
-          includes(['cdn_basic', 'cdn-basic-free'], planCode),
+        const addonPlan = find(addonPlans, ({ planCode }) =>
+          includes(['cdn-basic', 'cdn-basic-free'], planCode),
         );
 
         const { data } = await HostingCdnOrderService.simulateCartForUpgrade(
-          cartId,
           serviceName,
           addonPlan,
           serviceId,
         );
 
-        return { cart: data.order, cartId, addonPlan, serviceId };
+        return { cart: data.order, cartId: null, addonPlan, serviceId };
```

Comment:
```suggestion
        return { cart: data.order, addonPlan, serviceId };
```
there's no point returning the `cartId` if it's set to `null`